code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -451,6 +452,8 @@ static HookFunction initFunction([]()
 	fx::ScriptEngine::RegisterNativeHandler("GET_VEHICLE_WHEELIE_STATE", readVehicleMemory<unsigned char, WheelieState>);
 	fx::ScriptEngine::RegisterNativeHandler("SET_VEHICLE_WHEELIE_STATE", writeVehicleMemory<unsigned char, WheelieState>);
 
+	fx::ScriptEngine::RegisterNativeHandler("GET_TRAIN_CURRENT_TRACK_NODE", readVehicleMemory<int, TrainTrackNodeIndex>);
```

Comment:
CVehicle doesn't go all the way to 0x14C0, please add some sort of vehicle type assertion and return, a sane default if not.

Actually, why don't the things for CBike above do the same?!

Reply:
Fixed.